test_sets/External_Consent_Status_ON.txt

```diff
@@ -0,0 +1,33 @@
+./test_scripts/Polices/External_UCS/External_Consent_ON/006_ATF_Policies_External_Consent_ON_disallowed_by_external_consent_entities_off_user_consent_disallowed.lua Check_that_SDL_returns_USER_DISALLOWED_when_disallowed_by_external_consent_entities_on_exists_externalConsentStatus_ON_and_functional_group_disallowed_by_user
+./test_scripts/Polices/External_UCS/External_Consent_ON/007_ATF_Policies_External_Consent_ON_disallowed_by_external_consent_entities_omitted_data_consent_disallowed.lua Check_that_SDL_returns_DISALLOWED_when_disallowed_by_external_consent_entities_on/off_omitted_externalConsentStatus_ON_and_data_consent_disallowed
+./test_scripts/Polices/External_UCS/External_Consent_ON/008_ATF_Policies_External_Consent_ON_disallowed_by_external_consent_entities_omitted_user_consent_allowed.lua Check_that_SDL_returns_SUCCESS_when_disallowed_by_external_consent_entities_on/off_omitted_externalConsentStatus_ON_and_functional_group_allowed_data_consent_allowed
+./test_scripts/Polices/External_UCS/External_Consent_ON/009_ATF_Policies_External_Consent_ON_disallowed_by_external_consent_entities_omitted_user_consent_disallowed.lua Check_that_SDL_returns_USER_DISALLOWED_when_disallowed_by_external_consent_entities_on/off_omitted_externalConsentStatus_OFF_and_functional_group_disallowed_by_user
+./test_scripts/Polices/External_UCS/External_Consent_ON/010_ATF_Policies_External_Consent_ON_disallowed_by_external_consent_entities_omitted_user_consent_omitted.lua Check_that_SDL_returns_USER_DISALLOWED_when_disallowed_by_external_consent_entities_on/off_omitted_externalConsentStatus_ON_and_functional_group_omitted
+./test_scripts/Polices/External_UCS/External_Consent_ON/011_ATF_Policies_External_Consent_ON_through_ignition_cycles_allowed_RPCs.lua Check_that_SDL_uses_value_externalConsentStatus_ON_when_disallowed_by_external_consent_entities_on_exists_allowed_RPCs_during_ign_cycles
+./test_scripts/Polices/External_UCS/External_Consent_ON/012_ATF_Policies_External_Consent_ON_through_ignition_cycles_user_disallowed_RPCs.lua Check_that_SDL_uses_value_externalConsentStatus_ON_when_disallowed_by_external_consent_entities_off_exists_during_ign_cycles
+./test_scripts/Polices/External_UCS/External_Consent_ON/013_ATF_Policies_External_Consent_ON_through_ignition_cycles_disallowed_by_external_consent_entities_omitted.lua Check_that_SDL_uses_value_externalConsentStatus_ON_when_disallowed_by_external_consent_entities_on_exists_user_disallow_RPCs_during_ign_cycles
+./test_scripts/Polices/External_UCS/External_Consent_ON/014_ATF_Policies_External_Consent_ON_disallowed_change_to_OFF_allowed.lua Check_that_functional_grouping_changes_to_user_allowed_and_SDL_processes_RPC_when_externalConsentStatus_changes_to_OFF
+./test_scripts/Polices/External_UCS/External_Consent_ON/015_ATF_Policies_External_Consent_ON_external_consent_disallowed_user_consent_allowed.lua Check_that_SDL_processes_RPCs_when_consentedFunctions_true_externalConsentStatus_ON_precondition_user_allowed
```

Comment:
@dboltovskyi, as I understand precondition should be user_disallowed. Please clarify.